src/db/sysdb_search.c

```diff
@@ -242,6 +242,7 @@ int sysdb_getpwnam(TALLOC_CTX *mem_ctx,
     }
```

Comment:
What about assigning "ret = EOK" after "non-fatal" comment?
Not sure whether will cause "set but not read" in some cases.  (aka dead assignment(

Reply:
I thought about that, @lslebodn ... but I noticed that in a few other parts of the same file we add the ret = EOK just before the label, thus I've mimic the other parts and decided to go for this approach.

With that in mind, would you prefer me to change? If yes, please, let me know and I'll re-submit the patch.

Reply:
Well, I've changed according to your request as I don't have any strong preference on this one.

Reply:
I think in this case @lslebodn's request makes sense because we are explicitly recovering from a failure.